L1Trigger/TrackTrigger/src/TrackQuality.cc

```diff
@@ -86,18 +83,31 @@ std::vector<float> TrackQuality::featureTransform(TTTrack<Ref_Phase2TrackerDigi_
       tmp_trk_nlaymiss_interior++;
   }
 
-  float eta = abs(aTrack.eta());
-  int eta_size = static_cast<int>(eta_bins.size());
-  // First iterate through eta bins
-
-  for (int j = 1; j < eta_size; j++) {
-    if (eta < eta_bins[j] && eta >= eta_bins[j - 1])  // if track in eta bin
-    {
-      // Iterate through hitpattern binary
-      for (int k = 0; k <= 6; k++)
-        // Fill expanded binary entries using the expected hitmap table positions
-        hitpattern_expanded_binary[hitmap[j - 1][k]] = hitpattern_binary[k];
-      break;
+  if (useHPH) {
+    double tmp_trk_tanL = aTrack.tanL();
```

Comment:
As these variables are used later we can define them outside the (useHPH) block and use them again in the feature_map

Reply:
Done